src/utils.js

```diff
@@ -52,4 +52,17 @@ const isEmpty = (list) => list.length === 0;
 const updateData = (data, update) => data.map((item) => item.id === update.id ? update : item);
 const updateItem = (item, prop) => ({ ...item, ...prop });
 
-export { DateFormat, formatDate, countDuration, capitalizeFirstLetter, isEmpty, updateData, updateItem, sortEvents };
+const now = dayjs();
+
+const filter = {
+  [FilterTypes.EVERYTHING]: (events) => events,
```

Comment:
Лучше создавать копию, чтобы случайно не мутировать оригинальный массив 
```suggestion
  [FilterTypes.EVERYTHING]: (events) => [...events],
```